sdk/identity/azure-identity-broker/azure/identity/broker/_browser.py

```diff
@@ -52,8 +52,9 @@ class InteractiveBrowserBrokerCredential(_InteractiveBrowserCredential):
         may still log in with a different username.
     :paramtype cache_persistence_options: ~azure.identity.TokenCachePersistenceOptions
     :keyword int timeout: seconds to wait for the user to complete authentication. Defaults to 300 (5 minutes).
-    :keyword int parent_window_handle: If your app is a GUI app running on a modern Windows system, you are required to
-        also provide its window handle so that the sign in UI window will properly pop up on top of your window.
+    :keyword int parent_window_handle: If your app is a GUI app running on a modern Windows or Mac system, you
```

Comment:
Let's avoid the word "modern", since it means different things to different people.
```suggestion
    :keyword int parent_window_handle: If your app is a GUI app running on Windows 10+ or macOS, you
```